plugins/omelasticsearch/omelasticsearch.c

```diff
@@ -232,14 +232,15 @@ static rsRetVal curlSetup(wrkrInstanceData_t *pWrkrData);
 BEGINcreateInstance
 CODESTARTcreateInstance
 	pData->fdErrFile = -1;
-	pthread_mutex_init(&pData->mutErrFile, NULL);
+	CHKiRet(pthread_mutex_init(&pData->mutErrFile, NULL));
```

Comment:
pthread_mutex_init() does not return a rsyslog error code. So this should be mapped. It's not a merge stopper, but I'll leave it open a bit more so that you can fix it. Otherwise I'll do after merge.